packages/react-components/react-data-grid-react-window/package.json

```diff
@@ -32,9 +34,11 @@
     "@fluentui/react-theme": "^9.1.5",
     "@fluentui/react-utilities": "^9.3.0",
     "@griffel/react": "^1.4.2",
+    "react-window": "^1.8.6",
     "tslib": "^2.1.0"
   },
   "peerDependencies": {
+    "@fluentui/react-components": "^9.7.4",
```

Comment:
Should be version there fixed? 😕 As once `DataGrid` goes to stable -> imports will be broken as they are from `/unstable`.

Reply:
honestly I think a caret is the better option, because that will reduce duplicates in node_modules. We are allowed to break `/unstable` imports.

Reply:
Will solve this as a part of #26297 to avoid blowing up this PR